lit-element.js

```diff
@@ -202,8 +210,15 @@ class DwLitElement extends PolymerLitElement {
   }
 
   updated(changedProps) {
+    if (!config || (config && config.disabled)) {
```

Comment:
@nirmalbaldaniya-dream As we haven't changed the behavior for the `config.disabled`, we don't need to write a single line corresponding to it.

As per the last meeting only behavior we are going to change was a configuration property `config.debugRender`.